src/constants/general.js

```diff
@@ -1,38 +1,36 @@
-// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
 // See License.txt for license information.
 
-import keyMirror from 'utils/key_mirror';
+export default {
+    CONFIG_CHANGED: 'config_changed',
 
-const GeneralTypes = keyMirror({
-    RECEIVED_APP_STATE: null,
-    RECEIVED_APP_CREDENTIALS: null,
-    REMOVED_APP_CREDENTIALS: null,
-    RECEIVED_APP_DEVICE_TOKEN: null,
+    PROFILE_CHUNK_SIZE: 100,
+    CHANNELS_CHUNK_SIZE: 50,
```

Comment:
If we have a file for `Posts` constants that include the `POSTS_CHUNK_SiZE` why not add a file for users (profiles and channels for this two?